index.md

```diff
@@ -53,10 +57,11 @@ Depending on what kind of work you are trying to do, you may wish to engage with
 * **Blip**: You may wish to build Blip, our web application, locally but still run against our hosted back-end services. If so, check out the README in the [blip](https://github.com/tidepool-org/blip) repository. (Of course you can also just run the production Blip at [blip.tidepool.org](https://blip.tidepool.org).
 * **Your Applications**: You may wish create your own application that accesses our hosted back-end services (the Tidepool Platform). If it's a web app, you may wish to look at Blip. If it's a mobile app, check out the [Nutshell](https://github.com/tidepool-org/nutshell-ios) or Blip Notes repos (a.k.a. [Urchin for Android](https://github.com/tidepool-org/urchin-android) or [Urchin for iOS](https://github.com/tidepool-org/urchin)).
   * Please don't develop your app against our production environment without talking to us. Instead, please use our integration environment, found at int-blip.tidepool.org and int-api.tidepool.org.
-  * Read our [data model documentation](/data-model/) to understand what medical data we ingest and how it’s organized.
-  * [These comments](https://github.com/tidepool-org/tide-whisperer/blob/master/tide-whisperer.go#L193) show documentation for our data APIs or you can help us improve our [draft API documentation](http://developer.tidepool.io/tidepool-api/index/).
+  * Read our [data model documentation](http://developer.tidepool.io/data-model/) to understand what medical data we ingest and how it’s organized.
+  * [These comments](https://github.com/tidepool-org/tide-whisperer/blob/master/tide-whisperer.go#L193) show some documentation for our data APIs.
 
 ## Communication
+
 We are currently using [this Gitter channel](https://gitter.im/tidepool-org/public) for communication with the open-source community, although we plan to move to a public Slack channel as soon as that option is available. **We will be the first to admit that we aren't as responsive as we could or should be in Gitter.** We are very grateful to community members who have been so generous with their time supporting other members.
```

Comment:
As I mentioned yesterday in stand-up, I'm not going to make any changes to this text since there seems to be some stuff to be discussed given @pazaan and @darinkrauss's comments. If/when you all decide on an update to our community outreach via Gitter policy, please do a PR to update the text.